src/data.upgrade.func.php

```diff
@@ -38,8 +38,8 @@ function litespeed_update_4() {
 	$q = 'DROP TABLE IF EXISTS ' . $tb;
 	$wpdb->query( $q );
 
-	if ( file_exists( LITESPEED_STATIC_DIR . '/ccss' ) ) {
-		File::rrmdir( LITESPEED_STATIC_DIR . '/ccss' );
+	if ( file_exists( LITESPEED_STATIC_DIR . '/ccsjs' ) ) {
```

Comment:
This is a good catch. However it won't work for v4 existing users. Can put this into v4.3 upgrade func.

Reply:
Done in https://github.com/litespeedtech/lscache_wp/commit/9d7baa0bc799792c22b352c976eb83184c74f8c1